manifests/plugin/openscap.pp

```diff
@@ -0,0 +1,63 @@
+# = Foreman OpenSCAP plugin
+#
+# This class installs OpenSCAP plugin
+#
+# === Parameters:
+#
+# $configure_openscap_repo::  Enable custom yum repo with packages needed for foreman_openscap,
+#                             type:boolean
+#
+# $scap_client_module_dir::   if you set a directory here, installer will upload foreman_scap_client
+#                             puppet module into this directory for you
+#                             e.g. /etc/puppet/environments/production/modules
+#
+class foreman::plugin::openscap (
+  $configure_openscap_repo = false,
```

Comment:
I wanted to keep consistency with other custom repos (e.g. https://github.com/theforeman/puppet-foreman/blob/master/manifests/params.pp#L218). Also I think user should explicitly allow extra repo. Maybe a better parameter documentation would help?